lib/payment/response_classes/merchant_stripe.rb

```diff
@@ -57,8 +57,8 @@ class LegalEntity < StripeHash
     include DobConvertible
 
     attr_accessor :additional_owners, :address, :business_name,
-                  :business_tax_id, :dob, :first_name, :last_name, :type,
-                  :verification
+                  :business_tax_id, :business_tax_id_provided, :dob,
+                  :first_name, :last_name, :type, :verification
```

Comment:
business_tax_id_provided is just a flag, say if we previously provided the business_tax_id.

business_tax_id is a little different, we can set it, but never get back the value (only the business_tax_id_provided flag)